app/controllers/standup_meetings/completions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class StandupMeetings::CompletionsController < ApplicationController
+  def create
+    @standup_meeting = StandupMeeting.find(params[:standup_meeting_id])
```

Comment:
should be able to just `include` the `standup_meeting_group` to this initial call.